_test/test/serve_integration_test.dart

```diff
@@ -135,6 +138,8 @@ void main() {
       'web',
       '--build-filter',
       'web/sub/main.dart.js',
+      '--verbose',
```

Comment:
Trying to make the server emit verbose messages

Reply:
How about we go with https://github.com/dart-lang/build/pull/3145 and enable this more centrally.

Reply:
removed.